packages/js/src/plugins/auctionHouseModule/errors.ts

```diff
@@ -134,10 +149,28 @@ export class WithdrawFromBuyerAccountRequiresSignerError extends AuctionHouseErr
     super({
       options,
       key: 'withdraw_from_buyer_account_requires_signer',
-      title: 'Withdraw From Buyer Account Requires Signer',
+      title: 'Withdraw From Buyer Account Requires Signer.',
       problem:
         'You are trying to withdraw from buyer account without providing a signer.',
       solution: 'Either a buyer or authority must be a Signer.',
     });
   }
 }
+
+/** @group Errors */
+export class PartialPriceMismatchError extends AuctionHouseError {
+  constructor(options?: MetaplexErrorOptions) {
+    super({
+      options,
+      key: 'partial_price_mismatch_signer',
+      title:
+        'The calculated partial price does not equal the partial price provided.',
+      problem:
+        'You are trying to execute the sale by providing the wrong price for the token.',
```

Comment:
Could we pass in the expected price and the actual price as error arguments so we can show them in the `problem` text? You can use `formatAmount` to display them as text.